app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class TournamentsController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+
+  def index
+  end
+
+  def create
+    @company_name = params[:company_name]
```

Comment:
There's no need to use an instance variable here
The main reason to use instance variables is when there's server side rendering, and the view renders an instance variable
Otherwise it causes problems in my opinion - you're sharing state that can be modified by anyone
So I don't think you need these instance variables